src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -1069,6 +1069,7 @@ def buildImage(config, image) {
     if ("${env.build_dockers}" == "true") {
         config.logger.info("Forcing building file per user request: ${filename} ... ")
         need_build++
+        extra_args += ' --no-cache'
```

Comment:
why not to add it to project yaml file as part of ```image.build_args``` section? why need to have it global?

Reply:
OK, though the problem is pretty general, I'm solving particular project use case. I've created a PR to a project. Closing this one...